src/sinks/sematext.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{
+    sinks::elasticsearch::ElasticSearchConfig,
+    sinks::util::Compression,
+    topology::config::{DataType, SinkConfig, SinkContext},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct SematextConfig {
+    cloud: Cloud,
+    #[serde(flatten)]
+    inner: ElasticSearchConfig,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum Cloud {
+    NorthAmerica,
+    Europe,
+}
+
+#[typetag::serde(name = "sematext")]
+impl SinkConfig for SematextConfig {
+    fn build(&self, cx: SinkContext) -> crate::Result<(super::RouterSink, super::Healthcheck)> {
+        let mut es_config = self.inner.clone();
+
+        let host = match &self.cloud {
+            Cloud::NorthAmerica => "https://logsene-receiver.sematext.com",
+            Cloud::Europe => "https://logsene-receiver.sematext.com",
```

Comment:
Should be logsene-receiver.eu.sematext.com, @LucioFranco

Reply:
Oh wow! Yup, good catch! Will get this updated.